src/app/shared/material/material.module.ts

```diff
@@ -12,6 +12,10 @@ import { MatMenuModule } from '@angular/material/menu';
 import { MatSnackBarModule } from '@angular/material/snack-bar';
 import { MatCardModule } from '@angular/material/card';
 import { MatSelectModule } from '@angular/material/select';
+import {MatTabsModule} from '@angular/material/tabs';
```

Comment:
please follow general style guides use {ImportName} or { ImportName } but not both